app/catalog/page.tsx

```
@@ -0,0 +1,259 @@
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      className="ml-2 text-sm text-primary hover:underline"
+                    >
+                      @{selectedDeveloper.username}
+                    </a>
+                  </CardTitle>
+                  <p className="text-foreground">{selectedDeveloper.bio}</p>
+                  <div className="flex flex-wrap gap-4 mt-2 text-sm text-muted-foreground">
+                    {selectedDeveloper.location && (
+                      <span className="flex items-center">
+                        <svg xmlns="http://www.w3.org/2000/svg" className="h-4 w-4 mr-1" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)